databricks/koalas/generic.py

```diff
@@ -1424,7 +1428,7 @@ def kurtosis(spark_column, spark_type):
     kurt = kurtosis
 
     def min(
-        self, axis: Union[int, str] = None, numeric_only: bool = None
+        self, axis: Union[int, str] = None, numeric_only: bool = True
```

Comment:
This should remain as `None`, right?

Reply:
Right. Thanks for the correction

Reply:
Maybe I think we can use `True` for pandas compatibility?? I just matched it to pandas'. 

for `sum`, `prod`, `mix`, `max`, they uses `True` as a default value for `numeric_only`.

In addition, we can apply same logic for `numeric_only` for all methods - `numeric_only = None if axis == 1 and numeric_only is True else numeric_only` - if we use same default value.

WDYT?

Reply:
pandas uses `None` as a default value, right?
e.g., https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.sum.html

Reply:
Oops, sorry for annoying. It was my misread. I'll address it  😅

Reply:
Addressed it. Thanks, @ueshin